Doc/library/dis.rst

```diff
@@ -398,13 +401,17 @@ The Python compiler currently generates the following bytecode instructions.
    Push the *i*-th item to the top of the stack. The item is not removed from its
    original location.
 
+   The stack is indexed from 1, so COPY 1 will copy TOS.
```

Comment:
Maybe instead of adding this, we could define ``TOS(i)`` at the top along with TOS1, TOS2 etc, and then reword the definition here to use those.

Reply:
I tried something along those lines in my last commit let me know if you like it better this way.